django/contrib/admin/checks.py

```diff
@@ -3,11 +3,15 @@
 
 from itertools import chain
 
-from django.contrib.admin.utils import get_fields_from_path, NotRelationField, flatten
+from django.contrib.admin.utils import (
```

Comment:
Surely no much difference in readability and 80 doesn't look "significantly uglier" ;-)